heroku/cli
# refactor(container-registry-v5): move container:run to CLI & upgrade to oclif

### Description

This converts `container:run` to oclif and updates its associated test.

### Testing

1) Create an app based on a Dockerfile with a web process
2) Run `./bin/run container:run web bash -a APP_NAME` with this branch pulled down (it can be any command, not just `bash`)
3) Verify the command succeeds with the bash prompt within the container shows up
4) Compare the results with prod cli command and make sure things look 👍
5) Repeat steps 2-4 but without a command argument (such as `bash`) included (e.g., `./bin/run container:run web -a APP_NAME`)
